content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst

```diff
@@ -0,0 +1,146 @@
+
+To create a new mailing list, click :guilabel:`Create` in the upper left corner of the
+:guilabel:`Mailing Lists` page. Doing so, will reveal a blank mailing list template pop-up.
+
+.. image:: mailing_lists_blacklists/sms-mailing-list-popup.png
+   :align: center
+   :alt: View of the mailing list pop-up window in Odoo SMS Marketing.
+
+First, choose a name for the :guilabel:`Mailing List`, and activate the option :guilabel:`Is
+Public`, to make the mailing list accessible to recipients on the :guilabel:`Subscription Management
+page`. Doing so allows users to update their subscription preferences at any time.
+
+.. note::
+   Checking the :guilabel:`Is Public` box is not required, but is recommended to maintain good
+   customer relations.
+
+When those fields are filled in, click :guilabel:`Create`, and Odoo removes the pop-up. Then, Odoo
+returns to the main :guilabel:`Mailing Lists` page, where there is now a new, accessible list for
+the mailing list that was just created.
+
+As mentioned before, to edit (and customize) the mailing list further, select the mailing list from
+the main :guilabel:`Mailing Lists` page, and Odoo reveals the mailing list detail form.
+
+Along the top of the mailing list detail form, there are various analytical smart buttons displaying
+statistics for different metrics related to the mailing list (e.g. :guilabel:`Recipients`,
+:guilabel:`Mailings`, etc.).
+
+To take a closer look at (or make changes to) any of those elements, click the desired smart button,
+and Odoo reveals a separate page with in-depth data associated with the mailing list.
+
+To make changes to the mailing list itself, click the :guilabel:`Edit` button in the upper-left
+corner of the mailing list detail form.
+
+.. image:: mailing_lists_blacklists/sms-mailing-list.png
+   :align: center
+   :alt: View of the mailing list template form in Odoo SMS Marketing.
+
+.. note::
+   Don't forget to hit the :guilabel:`Save` button once changes have been made.
+
+Mailing lists contacts
+======================
+
+If :guilabel:`Mailing List Contacts` is selected from the :guilabel:`Mailing Lists` header drop-down
+menu, Odoo reveals a separate page with all the contacts associated with one (or more) of the
+configured mailing lists in the database.
+
+.. image:: mailing_lists_blacklists/mailing-list-contacts-page.png
+   :align: center
+   :alt: View of the mailing lists contact page in the Odoo SMS Marketing application.
+
+.. note::
+   By default, Odoo reveals the :guilabel:`Mailing List Contacts` page with the :guilabel:`Exclude
+   Blacklisted Phone` filter in the search bar. Therefore only showing contact information for
+   recipients who still want to receive communications and mailings.
+
+Communication history in the Chatter
+------------------------------------
+
+An accessible record of the sent mailings is kept on each recipient's :guilabel:`Chatter`, located
+beneath a recipient's contact form (in the :guilabel:`Contacts` application).
```

Comment:
Don't think these get guilabels since it's not active instruction on the UI.
```suggestion
An accessible record of every sent mailing(s) is kept on each recipient's *chatter* section, located
beneath a recipient's contact form (in the *Contacts* application).
```